gapis/service/service.proto

```diff
@@ -1329,7 +1329,27 @@ message ProfilingData {
     repeated Group groups = 3;
   }
 
+  message GpuCounters {
+    message Counter {
+      int32 trackId = 1;
+      uint64 ts = 2;
+      double value = 3;
+    }
+
+    message Track {
+      uint32 id = 1;
+      string name = 2;
+      string description = 3;
+      string unit = 4;
+      bool default = 5;
+    }
+
+    repeated Counter counters = 1;
```

Comment:
Storing the counter data as arrays will be more efficient. So
- rename Track -> Counter
- Add a `repeated uint64 ts` field to Counter
- Add a `repeated double value` field to Counter